packages/web/locales/ja.json

```diff
@@ -0,0 +1,66 @@
+{
+  "language":"japanese",
+
+  "english":"英語",
+  "german":"ドイツ語",
+  "french":"フランス語",
+  "japanese":"日本語",
+
+  "options.language.h3":"言語",
+  "options.theme.h3":"テーマ",
+
+  "options.generation.h3":"Geometry Generation",
+  "options.generation.caching":"experimental geometry caching",
+  "options.generation.timeout":"timeout for solids generation",
+
+  "options.viewer.h3":"ビューアー",
+  "options.viewer.zoomonparamchange":"zoom to fit on new parameters",
+  "options.viewer.zoomonreload":"zoom to fit on design load",
+
+  "options.storage.h3": "ストレージ",
+  "options.storage.settingspath":"ストレージ パス",
+
+  "options.filehandling.h3":"ファイル処理",
+  "options.filehandling.converttoscript":"Convert external formats to JSCAD scripts",
+
+  "options.shortcuts.h3":"キーボード ショートカット",
+
+  "command":"コマンド",
+  "keybinding":"キーバインディング",
+  "when":"when",
+  "typeandenter":"type and hit enter",
+
+  "notsettable":"(not settable)",
+  "see":"see",
+
+  "grid":"グリッド",
+  "axes":"アクシス",
+  "autorotate": "自動回転",
+  "auto reload": "自動リロード",
+  "export":"エクスポート",
+  "load jscad project":"プロジェクトロード",
+
+  "STL (Binary)":"STL (バイナリ)",
+  "STL (ASCII)":"STL (アスキー)",
+  "AMF (experimental)":"AMF",
+
+  "update":"アップデート",
+  "instant update":"自動アップデート",
+  "reset":"リセット",
+
+  "toPresetView":"Reset view",
+  "front":"前面",
+  "back":"背面",
+  "top":"上面",
+  "bottom":"下面",
+  "left":"左側",
+  "right":"右側",
+
+  "setProjectionType":"View type",
+  "perspective":"",
+  "orthographic":"",
+
+  "reEvaluateCode":"Reload script",
+
+  "always":""
+}
```

Comment:
hmm is the language prefix for japanese not **jp** and no **ja** (I mean for the file name ja.json)

Reply:
Nope. It’s ‘ja’

https://en.m.wikipedia.org/wiki/List_of_ISO_639-1_codes

Reply:
ok, I stand corrected :)